_protocols/beer-dna-extraction.md

```diff
@@ -23,81 +23,110 @@ Needed consumables
+    Shake it carefully the Erlenmeyer flask to remove the CO2. A foam will form, whose the size depends on the beer, its temperature and for how long it was open.
+
+    ![](/images/protocols/beer-dna-extraction/erlenmeyer_with_beer.svg){: width="35%"}
+
+3. Transfer the beer (not the foam) into 50ml Falcon tubes using a large pipette
+
+    ![](/images/protocols/beer-dna-extraction/falcon_with_beer.svg){: width="75%"}
    
-4. Put the tubes (with the lids) in the centrifuge for 10min at full-speed and 4℃
+    - Make sure each tube gets the same quantity (to balance the centrifuge for the next step)
+    - Put a lid on each tube but don't close them until the next step (CO2 needs to be evacuated)
    
-   This step separates the liquid phase and the solid phase (which contains yeast among other things)
+4. Put the tubes (with the lids closed) in the centrifuge for 10min at full-speed and 4℃
+
+    Be careful that the centrifuge is correctly balanced: put same number of tubes on each opposite side.
    
-5. Transfer the pellet, solid part on the bottom, into 5ml Eppendorf tubes
-6. Mix with 1ml neutral buffer (tris/buffer 7.4 1M) with a pipette
-7. Label the tubes
-8. Put the tubes into the centrifuge one more time for about 5 min at full-speed
-9. Transfer the pellet into 4 5ml Eppendorf tubes
+    This step separates the liquid phase and the solid phase (which contains yeast among other things): 
+
+    ![](/images/protocols/beer-dna-extraction/after_centrifuge_1.svg){: width="25%"}
+
+5. Trough the liquid out
+6. Take 1ml of neutral buffer (tris/buffer 7.4 1M) with a pipette
+
+   ![](/images/protocols/beer-dna-extraction/buffer_collection.svg){: width="25%"}
+
+7. Add it to the pellet, solid part on the bottom, and mix it by resuspending the content (pull/push several times)
+
+   ![](/images/protocols/beer-dna-extraction/suspend_pellet.svg){: width="85%"}
 
-### Break the cell membranes of the yeasts
+6. Transfer into 5ml Eppendorf tubes
+
+   ![](/images/protocols/beer-dna-extraction/transfer_to_eppendorf.svg){: width="20%"}
+
+8. Label the tubes
+9. Put the tubes into the centrifuge one more time for about 5 min at full-speed
+10. Transfer the pellet into 4 5ml Eppendorf tubes
+
+### Break the cell walls of the yeasts and nucleus
+
+![](/images/protocols/beer-dna-extraction/yeast_cell.svg){: width="50%"}
+
+*A yeast cell - Frankie Robertson,  CC ASA, [Wikimedia](https://en.wikipedia.org/wiki/File:Yeast_cell_english.svg)*
 
-1. Put the tubes into an ethanol bath (-80℃)
 2. Add 200µl of lysis buffer into each tube and suspend them
 3. Repeat 2 times
-  1. Put the tubes into the ethanol bath for 2 min
-  2. Put the tubes in boiling water (100℃) for 2 min
-  3. Vortex the tubes for 30s
+    1. Put the tubes into the ethanol bath or dry ice for 2 min
+    2. Put the tubes in boiling water (100℃) for 2 min
+    3. Vortex the tubes for 30s
   
-### Extract the DNA from the cell
+### Extract the DNA from the content of the cell
```

Comment:
I think you extract DNA out of something, not out of others. I think the former was more correct, some for the sentence below.